src/config/config.ts

```diff
@@ -396,21 +396,36 @@ export class Config implements IConfig {
   }
 
   public scopedEnvVar(k: string): string | undefined {
-    return process.env[this.scopedEnvVarKey(k)]
+    return process.env[this.scopedEnvVarKeys(k).find(k => process.env[k]) as string]
   }
 
   public scopedEnvVarTrue(k: string): boolean {
-    const v = process.env[this.scopedEnvVarKey(k)]
+    const v = process.env[this.scopedEnvVarKeys(k).find(k => process.env[k]) as string]
     return v === '1' || v === 'true'
   }
 
+  /**
+   * @deprecated - use scopedEnvVarKeys instead which will account for bin aliases, this could be private/removed in a major
```

Comment:
I'm not sure how I feel about deprecating this since the overwhelming majority of oclif CLIs won't ever use bin aliases. For them it would be annoying to have to deal with an array when it's always going to be an array with one element in it

I wonder if it might be better to keep it un-deprecated and just document clearly in the jsdoc that if they're using bin aliases they should use the other method

What do you think?